server/views/admin-list-categories.hbs

```diff
@@ -1,6 +1,20 @@
 <div class="container">
-    <div class="row justify-content-md-center">
+     
+      <div class="row justify-content-md-center">
         {{#each category}}
+        {{#if this.hideCategory}}
```

Comment:
in the control panel - you should show it even if it's hidden (because this is an administrator) - just display a field that says "Hidden": true or false with the other card-texts